packages/tests-e2e/support/setup-test-framework.js

```diff
@@ -8,6 +8,7 @@ import { get } from 'lodash';
  * WordPress dependencies
  */
 import '@wordpress/jest-console';
+import '@wordpress/jest-puppeteer-axe';
```

Comment:
If we're not using it, questionable to be opting in now, vs. PR to be about only adding the new module. Can be fine to limit the size of the follow-up pull request, having no/minimal impact here.

Reply:
yes, will remove for now